manifests/agent/config.pp

```diff
@@ -28,7 +28,7 @@
       ensure  => file,
       mode    => '0600',
       path    => "${config_dir}/encryption.cfg",
-      content => template('check_mk/agent/encryption.cfg.erb'),
+      content => Sensitive(template('check_mk/agent/encryption.cfg.erb')),
```

Comment:
would this be a good situation to switch to epp?

Reply:
oh yeah, I said I'd look at that :)  It's still WIP, and to start with I wanted to figure out how to get the new acceptance tests to go green.

Reply:
Fixed.